superset/utils/machine_auth.py

```diff
@@ -58,17 +66,52 @@ def authenticate_webdriver(
         # Setting cookies requires doing a request first
         driver.get(headless_url("/login/"))
 
+        cookies = self.get_cookies(user)
+
+        for cookie_name, cookie_val in cookies.items():
+            driver.add_cookie({"name": cookie_name, "value": cookie_val})
+
+        return driver
+
+    def authenticate_browser_context(
+        self,
+        browser_context: BrowserContext,
+        user: User,
+    ) -> BrowserContext:
+        # Short-circuit this method if we have an override configured
+        if self._auth_webdriver_func_override:  # type: ignore
+            return self._auth_webdriver_func_override(browser_context, user)
```

Comment:
I wonder if this is correct - shouldn't this be
```suggestion
        # Short-circuit this method if we have an override configured
        if self._auth_browser_context_func_override:  # type: ignore
            return self._auth_browser_context_func_override(browser_context, user)
```